assets/js/modules/analytics-4/utils/report-rows-with-set-values.test.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Analytics-4 filter rows having (not set) values tests.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
```

Comment:
As above, and also fix the date:

```suggestion
 * Tests for utility to filter out Analytics report rows which don't have a value set.
 *
 * Site Kit by Google, Copyright 2024 Google LLC
```